include/znc/znc.h

```diff
@@ -89,6 +89,7 @@ class CZNC {
 	// generated through ZNC.
 	TrafficStatsMap GetTrafficStats(TrafficStatsPair &Users,
 			TrafficStatsPair &ZNC, TrafficStatsPair &Total);
+	TrafficStatsPair GetNetworkTrafficStats(const CString& sUsername, TrafficStatsMap& Networks);
```

Comment:
Any comments?
This function looks like a reverse of previous one (order of args/return)